Cython/Utility/ModuleSetupCode.c

```diff
@@ -737,7 +737,7 @@ static CYTHON_INLINE PyObject * __Pyx_PyDict_GetItemStrWithError(PyObject *dict,
   #define __Pyx_PyUnicode_GET_LENGTH(u)   PyUnicode_GET_LENGTH(u)
   #define __Pyx_PyUnicode_READ_CHAR(u, i) PyUnicode_READ_CHAR(u, i)
   #define __Pyx_PyUnicode_MAX_CHAR_VALUE(u)   PyUnicode_MAX_CHAR_VALUE(u)
-  #define __Pyx_PyUnicode_KIND(u)         PyUnicode_KIND(u)
+  #define __Pyx_PyUnicode_KIND(u)         ((int)PyUnicode_KIND(u))
```

Comment:
even though the Python docs state, that this will return an int, it actually returns a value of the underlying type of the PyUnicode_Kind enum. This could be e.g. an unsigned int, which causes warnings about signed/unsigned comparisions in some parts of the code.